nfc/labAttendancy.py

```diff
@@ -2,7 +2,11 @@
 import requests
 import json
 
-import config
+from tkinter import *
+import time
+
+
+
 
 def postData(data):
```

Comment:
`config.py` 読むのは残しておいてほしい
あと不要な改行は消してほしい

```suggestion
import json
from tkinter import *
import time

import config

def postData(data):
```